packages/server/src/queries/complex/pools/index.ts

```diff
@@ -114,13 +118,13 @@ export async function getPools(
       0.0 // Exact match
     );
 
-    // if not exact match for coinMinimalDenom or pool ID, search by poolNameByDenom (ex: OSMO/USDC)
+    // if not exact match for coinMinimalDenom or pool ID, search by poolNameByDenom (ex: OSMO/USDC) or coinName (ex: Bitcoin)
     if (coinMinimalDemonMatches.length > 0) {
```

Comment:
Why are we only searching names if coinMinimalDenom matches? Couldn't that not necessarily be the case? Or can `coinMinimalDenomMatches` var name be improved? I see it searches on `coinDenoms` and `id` not just `coinMinimalDenom`